### ISSUE TITLE
Token based 2fa driver

### ISSUE THREAD

**Issue by karmendra** [paschaldev/laravauth#6]
Thanks for this great package. This is a feature request.
Can we add additional driver for Token based 2fa login using Google Authenticator,
in past I used this package for [google 2fa](https://packagist.org/packages/pragmarx/google2fa-laravel). I want to give my user option to choose from EMAIL, SMS, or TOKEN based 2FA.

Thanks.

**Comment by karmendra** [comment]
I am not sure if you have enough time for doing it yourself, but if you could give me some basic guidance on where to start looking in code to add new 2fa, I will try and create it myself and submit a PR for this.

**Comment by paschaldev** [comment]
@karmendra 

Yeah work doesn't give me enough time. A PR is welcome... The code is pretty easy to grasp... OOP stuff.

**Comment by karmendra** [comment]
Ok, I will try my best to navigate my way thru the code and figure out how I can add this feature to this package, Thanks for a response.
